neuralhydrology/modelzoo/fc.py

```diff
@@ -70,6 +70,8 @@ def _get_activation(self, name: str) -> nn.Module:
             activation = nn.Tanh()
         elif name.lower() == "sigmoid":
             activation = nn.Sigmoid()
+        elif name.lower() == "linear":
+            activation = nn.Identity()
```

Comment:
Isn't this somewhat pointless? If there's only one layer, the activation won't be used anyways (see L38+/L50+), and multiple layers with identity activation are equivalent to one layer.

Reply:
We could add a check for the user calling for a deep linear model. Is this better than allowing the user to ask for it?